spec/features/user_sees_list_of_books_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature 'User sees a list of books' do
+  scenario 'successfully' do
+
+    books = create_list(:book, 3)
+
+    visit books_path
+
+    expect(page).to have_css('ul.books', :count => 1)
```

Comment:
I probably wouldn't test this because it's not that important. As long as the books are there, you're probably good to go!